examples/droplet/main.tf

```diff
@@ -1,3 +1,11 @@
+terraform {
+  required_providers {
+    digitalocean = {
+      source = "digitalocean/digitalocean"
```

Comment:
I'm not sure if the minimum version attribute is strictly required for this. This just downloads the most current version (2.8), it seems.

Reply:
I think it's a good idea to update this when examples are updated.